portfolio/src/main/webapp/styleOld.css

```diff
@@ -0,0 +1,250 @@
+/* General Style */
```

Comment:
I see this also has the "Old" suffix. Is it still being used?

Reply:
No, forgot to remove it in the last commit, will in the next.